RebbleOS/vibrate.c

```diff
@@ -48,70 +67,97 @@ void vibrate_init(void)
     _vibrate_queue = xQueueCreate(1, sizeof(uint8_t));
```

Comment:
No I was rightbefore the edit, you are passing a pointer to the queue, so is uint8_t correct? I suspect it is not passing a uint32_t